server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -517,17 +518,44 @@ const orderMutations = {
         include: [
           { association: 'paymentMethod' },
           { association: 'fromCollective', attributes: ['id', 'slug'] },
-          { association: 'collective', attributes: ['id', 'slug'] },
+          { association: 'collective', attributes: ['id', 'slug', 'HostCollectiveId'] },
         ],
       });
 
       // -- Some sanity checks to prevent issues --
-      const paymentMethodIds = uniq(orders.map(order => order.PaymentMethodId).filter(Boolean));
+      const isAddedFund = order =>
+        order.paymentMethod?.service === PAYMENT_METHOD_SERVICE.OPENCOLLECTIVE &&
+        order.paymentMethod.type === PAYMENT_METHOD_TYPE.HOST;
+      const paymentMethodIds = uniq(
+        orders
+          .filter(order => !isAddedFund(order))
+          .map(order => order.PaymentMethodId)
+          .filter(Boolean),
+      );
       const ordersIds = orders.map(order => order.id);
+      const addedFundOrders = orders.filter(order => isAddedFund(order));
+      const isUser = fromAccount?.type === types.USER;
+      const addedFundPaymentMethod = fromAccount
+        ? await models.PaymentMethod.findOne({
+            where: {
+              CollectiveId: fromAccount.id,
+              service: 'opencollective',
+              type: 'collective',
+              deletedAt: null,
+            },
+          })
```

Comment:
What's that?

Certainly not looking like a payment for added funds. 

It's collective to collective.

Reply:
The payment method for Added Funds is the one from the collective receiving funds.

I don't see what justify updating the payment method in this case.